src/Tools/dotnet-dump/Dumper.cs

```diff
@@ -49,7 +49,7 @@ public async Task<int> Collect(IConsole console, int processId, string output, b
                 output = Path.GetFullPath(output);
 
                 // Display the type of dump and dump path
-                string dumpTypeMessage = type == DumpTypeOption.Mini ? "minidump" : "minidump with heap";
+                string dumpTypeMessage = type == DumpTypeOption.Mini ? "dump" : "dump with heap";
```

Comment:
This is fine with me. Also I'd be fine if the code was simply:
````
string dumpTypeMessage = "dump";
````

I think the key piece of information in this print statement is the output path, not the content description.